db/seeds.rb

```diff
@@ -12,12 +12,3 @@
 Invoice.destroy_all
 Merchant.destroy_all
 Customer.destroy_all
-
-
```

Comment:
deleting these to avoid the seed loading problems?

Reply:
yes